log/log.go

```diff
@@ -64,25 +64,33 @@ var bufStdErr bytes.Buffer
 // that can be used for checking what the logger would've written. This is
 // mostly used for tests.
 func OutputToBuf() {
+	debugMut.Lock()
+	defer debugMut.Unlock()
 	stdOut = &bufStdOut
 	stdErr = &bufStdErr
 }
 
 // OutputToOs redirects the output of the log.*-outputs again to the os.
 func OutputToOs() {
+	debugMut.Lock()
+	defer debugMut.Unlock()
 	stdOut = os.Stdout
 	stdErr = os.Stderr
 }
 
 // GetStdOut returns all log.*-outputs to StdOut since the last call.
 func GetStdOut() string {
+	debugMut.Lock()
+	defer debugMut.Unlock()
 	ret := bufStdOut.String()
 	bufStdOut.Reset()
 	return ret
 }
 
 // GetStdErr returns all log.*-outputs to StdErr since the last call.
 func GetStdErr() string {
```

Comment:
This PR is exactly to do what you wrote so instead of having
```
require.True(t, strings.Contains(log.GetStdERrr(),"My Pony Error"))
```
we can have
```
require.True(t, log.Contains("My Pony Error"))
```
Since this is the only use case (now) for this PR, I think it makes sense... PLUS it's a bit more aesthetic!
What's your thoughts ?